tests/tensorflow/nightly20220314/nlp-wmt.libsonnet

```diff
@@ -55,12 +55,12 @@ local utils = import 'templates/utils.libsonnet';
     accelerator: tpus.v3_32,
```

Comment:
Can you remove these files that have everything in `configs` commented out? Remember to also remove them from targets.jsonnet, or the code won't compile.

Reply:
Done! Deleted files with zero configs.